src/main/java/it/gov/pagopa/reward/notification/service/feedback/RewardNotificationFeedbackMediatorServiceImpl.java

```diff
@@ -99,19 +103,28 @@ protected Mono<List<RewardOrganizationImport>> execute(List<StorageEventDto> pay
         return Flux.fromIterable(payload)
                 // consider just organization feedback upload events
                 .filter(this::isOrganizationFeedbackUploadEvent)
-                .doOnNext(i->log.info("[REWARD_NOTIFICATION_FEEDBACK] Processing import request: {}", i.getSubject()))
+                .doOnNext(i -> log.info("[REWARD_NOTIFICATION_FEEDBACK] Processing import request: {}", i.getSubject()))
                 .map(mapper)
                 // trace import request
                 .flatMap(importsRepository::createIfNotExistsOrReturnEmpty)
                 // retrieve and elaborate csv
                 .flatMap(this::retrieveAndElaborateCsv)
                 // finalize import request state and store it
                 .flatMap(this::finalizeImportRequest)
-                .doOnNext(i->log.info("[REWARD_NOTIFICATION_FEEDBACK] Import request processing results stored: {}", i.getFilePath()))
+                // TODO check if correct
+                .flatMap(i -> i.getStatus().equals(RewardOrganizationImportStatus.COMPLETE)
```

Comment:
```suggestion
                .flatMap(i -> RewardOrganizationImportStatus.COMPLETE.equals(i.getStatus())
```

Reply:
Are you sure that the notification should be send only if COMPLETE?

An institution can upload a partial file, in those cases you are not sending the notification...